.gitignore

```diff
@@ -7,6 +7,8 @@
 /spec/reports/
 /tmp/
 
+/Gemfile.lock
```

Comment:
Why remove the Gemfile.lock ? It should be under vcs

Reply:
+1 on this. I think it'll be difficult to track what changed when the lockfile gets removed. There is a possibility in which malicious/bad code (or unintentional breaking change) got introduced in a minor or patch version, and could help as a security precaution.

I see that this was remove to simplify the GitHub tests. Would creating a new step to remove the lockfile before the ruby setup step help solve the initial issue?

Reply:
As discussed on another gem. Not committing the Gemfile.lock used to be the recommended thing, so when i saw it was extremely outdated I simply 🔥 it.

It's no longer recommended though, so I'll bring it back